dotcom-rendering/package.json

```diff
@@ -290,4 +287,4 @@
     "src/lib/content.d.ts": "make gen-schema && make gen-fixtures",
     "src/lib/index.d.ts": "make gen-schema && make gen-fixtures"
```

Comment:
Outside this PR's scope but do these file paths need updating too?

Reply:
I'm not sure, which tool uses these fields?

Reply:
`make gen-schema` runs `TJS.generateSchema` based on CAPIType and updates `src/model/json-schema.json`. There's a dependency on this command being run whenever the types are changed. Previously this dependency was often forgotten about which could cause the site to crash - we use [run-if-changed](https://www.npmjs.com/package/run-if-changed) to automate this.

Reply:
`make gen-fixtures` refreshes our fixture data for a similar reason